components/form/CheckboxGroup.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { useFormContext } from 'react-hook-form';
+import PropTypes from 'prop-types';
+import { CheckboxLabel, ErrorLabel } from '.';
+
+export function Checkbox({ ...props }) {
+  const { register } = useFormContext();
+  return (
+    <>
+      <div className={`flex items-center ${props.className}`}>
```

Comment:
The `className` used here is ambiguous. If we want to add to add className then it should cover the containerClassName, inputClassName and labelClassName. 

In this case, the `props.className` used here should be `containerClassName`, it should be destructured above with others and defined in the `PropTypes` and `defaultProps`